src/main/java/de/uni_tuebingen/ub/nppm/db/DeleteDB.java

```diff
@@ -0,0 +1,35 @@
+package de.uni_tuebingen.ub.nppm.db;
+
+import static de.uni_tuebingen.ub.nppm.db.AbstractBase.getSession;
+import org.hibernate.Session;
+import org.hibernate.query.NativeQuery;
+
+public class DeleteDB extends AbstractBase {
+
+    public static void updateAttribute(String table, String attribute, int id) throws Exception {
+        try ( Session session = getSession()) {
+            session.getTransaction().begin();
+            NativeQuery query = session.createNativeQuery("UPDATE " + table + " SET " + attribute + "= :value WHERE ID= :id");
+            query.setParameter("value", null);
```

Comment:
Seems like you just support value to be hardcoded null, which doesn't make much sense to me in a generic function. You should add a `value` parameter after `attribute` that can be null, and let the caller decide.

Reply:
And in fact i think you should change the function naming & signature

I also think we already talked about this 1-2 weeks before. The main implementation should be in an additional function:

`public static void update(String table, int id, Map<String,String> columnsAndValues);`

And this function should only redirect with an implementation similar to this one:

```
public static void update(String table, int id, String column, String value) {
    update(table, id, Map.ofEntries(entry(column, value)));
}
```

Reply:
Done look at commit: problem DeleteDB solved